src/languages/es.ts

```diff
@@ -2044,6 +2044,8 @@ export default {
                     'Cada vez que se pague un informe utilizando Expensify ACH, se creará el correspondiente pago de la factura en la cuenta de Xero indicadas a continuación.',
                 xeroBillPaymentAccount: 'Cuenta de pago de las facturas de Xero',
                 xeroInvoiceCollectionAccount: 'Cuenta de cobro de las facturas Xero',
+                invoiceAccountSelectDescription:
```

Comment:
nit, for brewity should we name the key as 

```suggestion
                invoiceAccountSelectorDescription:
```